docker/base/start-examples.txt

```diff
@@ -30,18 +30,18 @@ Map a directory for direct file reading:
    - start firefly with - http://localhost/firefly/
 
 Write to logging directory outside of docker image:
-   > docker run -p 80:8080 -v /local/myLogDir:/usr/local/tomcat-base/logs --memory=8g --rm ipac/firefly:latest
+   > docker run -p 80:8080 -v /local/myLogDir:/firefly/logs --memory=8g --rm ipac/firefly:latest
    - start firefly with - http://localhost/firefly/
 
 Debugging:
-   > docker run -p 8055:8080 -p 5050:5050 -p 9050:9050 --memory=4g -e "ADMIN_PASSWORD=myPassword" -e DEBUG="TRUE" --rm --name firefly ipac/firefly:latest
+   > docker run -p 8055:8080 -p 5050:5050 --memory=4g -e "ADMIN_PASSWORD=myPassword" -e DEBUG="TRUE" --rm --name firefly ipac/firefly:latest
    - start firefly with - http://localhost:8055/firefly/
 
 Production like:
-   > docker run -p 8055:8080 -p 9050:9050 --memory=30g -e "ADMIN_PASSWORD=myPassword" -e -e DEBUG="FALSE" --name productionServer ipac/firefly:latest
+   > docker run -p 8055:8080 -p --memory=30g -e "ADMIN_PASSWORD=myPassword" -e DEBUG="FALSE" --name productionServer ipac/firefly:latest
    - start firefly with - http://localhost:8055/firefly/
 
 Production like, multiple tomcats, using a shared work directory:
-   > docker run -p 8055:8080 -p 9050:9050 --memory=30g -e "ADMIN_PASSWORD=myPassword" \
-             -e "FIREFLY_SHARED_WORK_DIR=/local/shared/work/area" -v /local/shared/work/area --name productionServer ipac/firefly:latest
+   > docker run -p 8055:8080 -p --memory=30g -e "ADMIN_PASSWORD=myPassword" \
+             -v /local/shared/work/area:/firefly/shared-workarea --name productionServer ipac/firefly:latest
```

Comment:
should we mentions starting a redis server and pointing to it here. I know we did not in the pass, but it is clearer now what to do.

Reply:
> Should we mention starting a redis server and pointing to it here? I know we did not in the pass, but it is clearer now what to do.

I don't think so since redis is only needed when you have multiple instances of Firefly running.  These examples are for a single instance.
With multiple instances, you may need to introduce docker-compose or may be swarm.  Both, are out of our scope.

Reply:
ok